src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -469,6 +470,13 @@ export default {
 		}
 	},
 	methods: {
+		trackAdvancedFilters() {
+			this.$kvTrackEvent(
+				'Search',
```

Comment:
@egoroza This line needs to be lowercased. Otherwise, thanks for getting the tracking up to date based on our new standard.